test/foundry/zone/TestTransferValidationZoneOfferer.t.sol

```diff
@@ -2563,6 +2654,168 @@ contract TestTransferValidationZoneOffererTest is BaseOrderTest {
         return (orders, fulfillments, bytes32(0), 2);
     }
 
+    struct OrderAndFulfillmentInfra {
+        OfferItem[] offerArray;
+        ConsiderationItem[] considerationArray;
+        OrderComponents orderComponents;
+        Order[] orders;
+        Fulfillment[] fulfillments;
+    }
+
+    function _buildOrdersAndFulfillmentsMirrorOrdersFromFuzzArgs(
+        Context memory context
+    ) internal returns (Order[] memory, Fulfillment[] memory) {
+        OrderAndFulfillmentInfra memory infra = OrderAndFulfillmentInfra(
+            new OfferItem[](context.args.nonAggregatableOfferItemCount),
+            new ConsiderationItem[](context.args.nonAggregatableOfferItemCount),
+            OrderComponentsLib.empty(),
+            new Order[](context.args.nonAggregatableOfferItemCount * 2),
+            new Fulfillment[](context.args.nonAggregatableOfferItemCount * 2)
+        );
+
+        {
+            for (
+                uint256 i;
+                i < context.args.nonAggregatableOfferItemCount;
+                i++
+            ) {
+                // Mint an ERC721 to sell.
+                // TODO: REMOVE: Come back and figure out how to fuzz the offerer.
+                test721_1.mint(offerer1.addr, context.args.tokenId + i);
+
+                {
+                    // Create the OfferItem[] for it.
+                    infra.offerArray = SeaportArrays.OfferItems(
+                        OfferItemLib
+                            .fromDefault(SINGLE_721)
+                            .withToken(address(test721_1))
+                            .withIdentifierOrCriteria(context.args.tokenId + i)
+                    );
+
+                    // Create the ConsiderationItem[] the offerer expects.
+                    infra.considerationArray = SeaportArrays.ConsiderationItems(
+                        ConsiderationItemLib
+                            .fromDefault(ONE_ETH)
+                            .withRecipient(offerer1.addr)
+                            .withStartAmount(context.args.amount)
+                            .withEndAmount(context.args.amount)
+                    );
+
+                    // Build first restricted order components, remove conduit key.
+                    infra.orderComponents = OrderComponentsLib
+                    .fromDefault(VALIDATION_ZONE)
+                    .withOffer(infra.offerArray)
+                    .withConsideration(infra.considerationArray)
+                    // TODO: REMOVE: Come back and figure out how to fuzz the
+                    //               conduit key.
+                        .withConduitKey(bytes32(0))
```

Comment:
test infra only has one conduit set up; in other tests i believe we fuzz on "conduit: yes/no" and conditionally use `bytes32(0)` or `conduitKey[One]` (the "new" test base classes just call it `conduitKey`)

Reply:
Thanks. I was going to implement that pattern (as [here](https://github.com/ProjectOpenSea/seaport/pull/1016/files#diff-945414f958c6be186ee73df95b1ea91413b7fc93e9464c54f2565917ffd4f5deR1413)), but I also kind of wanted to play around to see if I could get a test that uses multiple conduits running